templates/compose/paymenter.yaml

```
@@ -0,0 +1,57 @@
+      interval: 10s
+      timeout: 1s
+      retries: 3
+    environment:
+      MYSQL_PASSWORD: ${SERVICE_PASSWORD_MYSQL}
+      MYSQL_ROOT_PASSWORD: ${SERVICE_PASSWORD_MYSQLROOT}
+      MYSQL_DATABASE: "paymenter"
+      MYSQL_USER: "paymenter"
+  cache:
+    image: redis:alpine
+    environment:
+      - 'REDIS_PASSWORD=${SERVICE_PASSWORD_64_REDIS}'    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)